snap/snapcraft.yaml

```diff
@@ -0,0 +1,142 @@
+name: edgex-device-usb-camera
+base: core22
+license: Apache-2.0
+adopt-info: metadata
+
+architectures:
+  - build-on: amd64
+  - build-on: arm64
+
+grade: stable
+confinement: strict
+
+slots:
+  edgex-secretstore-token:
+    interface: content
+    content: edgex-secretstore-token
```

Comment:
The `content` can be removed here, as it defaults to the slot name.

Reply:
Sure. But let's just leave it in for now.